src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -1,39 +1,84 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './NewCommentForm.scss';
 
-export const NewCommentForm: React.FC = () => (
-  <form className="NewCommentForm">
-    <div className="form-field">
-      <input
-        type="text"
-        name="name"
-        placeholder="Your name"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <input
-        type="text"
-        name="email"
-        placeholder="Your email"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <div className="form-field">
-      <textarea
-        name="body"
-        placeholder="Type comment here"
-        className="NewCommentForm__input"
-      />
-    </div>
-
-    <button
-      type="submit"
-      className="NewCommentForm__submit-button button"
-    >
-      Add a comment
-    </button>
-  </form>
-);
+import { NewComment } from '../../react-app-env';
+
+interface Props {
+  selectedPostId: number;
+  onAddComment: (newComment: NewComment) => void;
+}
+
+export const NewCommentForm: React.FC<Props> = ({
+  selectedPostId,
+  onAddComment,
+}) => {
+  const [newName, setNewName] = useState('');
+  const [newEmail, setNewEmail] = useState('');
+  const [newBody, setNewBody] = useState('');
+
+  const clearForm = () => {
+    setNewName('');
+    setNewEmail('');
+    setNewBody('');
+  };
```

Comment:
again, speaking of states, just like with movies, since all inputs relate to a single comment, make state an object, and adjust all interactions with it.